akka-http-backend/src/test/scala/com/softwaremill/sttp/akkahttp/AkkaHttpStreamingTests.scala

```diff
@@ -1,17 +1,28 @@
-package com.softwaremill.sttp.streaming
+package com.softwaremill.sttp.akkahttp
 
 import akka.NotUsed
 import akka.actor.ActorSystem
-import akka.stream.Materializer
+import akka.stream.{ActorMaterializer, Materializer}
 import akka.stream.scaladsl.Source
 import akka.util.ByteString
 import com.softwaremill.sttp.SttpBackend
-import com.softwaremill.sttp.akkahttp.AkkaHttpBackend
-import com.softwaremill.sttp.testing.streaming.{ConvertToFuture, TestStreamingBackend}
+import com.softwaremill.sttp.testing.ConvertToFuture
+import com.softwaremill.sttp.testing.streaming.{StreamingTest, TestStreamingBackend}
 
 import scala.concurrent.Future
 
-class AkkaHttpStreamingTests(actorSystem: ActorSystem)(implicit materializer: Materializer)
+class AkkaHttpStreamingTest extends StreamingTest[Future, Source[ByteString, Any]] {
```

Comment:
maybe this could be combined into a single class? That is, `StreamingTest` would define the abstract `backend`, `bodyProducer` etc fields

Reply:
Done